docs: add information on OHADA stock accounting

jniles opened the pull request:
This commit adds documentation on the OHADA stock accounting method for future reference.

@mbayopanda @jmcameron it's clear that I have to tackle this is small chunks.  Can you please give me a review on what I have so far?  Let me know if there are inaccuracies, or if the language is poorly chosen.

-----
[View rendered docs/en/stock-management/accounting/ifrs.md](https://github.com/jniles/bhima/blob/docs-add-accounting-information-to-stock-management-docs/docs/en/stock-management/accounting/ifrs.md)
[View rendered docs/en/stock-management/accounting/index.md](https://github.com/jniles/bhima/blob/docs-add-accounting-information-to-stock-management-docs/docs/en/stock-management/accounting/index.md)
[View rendered docs/en/stock-management/accounting/ohada.md](https://github.com/jniles/bhima/blob/docs-add-accounting-information-to-stock-management-docs/docs/en/stock-management/accounting/ohada.md)
[View rendered docs/en/stock-management/depot.md](https://github.com/jniles/bhima/blob/docs-add-accounting-information-to-stock-management-docs/docs/en/stock-management/depot.md)
[View rendered docs/en/stock-management/index.md](https://github.com/jniles/bhima/blob/docs-add-accounting-information-to-stock-management-docs/docs/en/stock-management/index.md)
[View rendered docs/en/stock-management/overview.md](https://github.com/jniles/bhima/blob/docs-add-accounting-information-to-stock-management-docs/docs/en/stock-management/overview.md)

jniles committed:
docs: add information on OHADA stock accounting

This commit adds documentation on the OHADA stock accounting method for
future reference.

mbayopanda commented on docs/en/stock-management/index.md:
```diff
@@ -9,6 +9,9 @@ hospitals.  Below you'll find an index to particular subsections that may be of
 2. [Inventories](./inventory.md)
 3. [Depots](./depot.md)
 4. [Stock Settings](./stock.settings.md)
+4. [Stock Accounting](./accounting/index.md)
```
The number here should not be `5.` ?

jniles committed:
docs(stock): added more information on IFRS

Adds more details on the IFRS stock accounting method for clarity.

jmcameron commented on docs/en/stock-management/accounting/index.md:
```diff
@@ -0,0 +1,163 @@
+in stock and creates an expense for the institution.
+
+These are the accounts used in the OHADA examples below:
+
```
I think it is worth noting briefly here that these are families of accounts (that is my understanding).  In the OHADA section, you could go into more detail, eg, why different sub-accounts are used, etc.

jniles replied:
Done.

I don't know that there is any rhyme or reason to the OHADA sub accounts and they sometimes change, so I'd prefer not to name specifics here.

jmcameron commented on docs/en/stock-management/accounting/index.md:
In the discussions you have led on this, you mentioned a couple of things that you may want to review here:
- At any point of time, you have to consult two accounts to figure out how much stock (value) you have (actual assets, right)?
- When does the 603 account get set to zero?

jniles replied:
I've addressed point 1.

The second point is a more general accounting procedure (you always wipe out income and expense accounts at the end of the fiscal period, no matter that accounting system) and I think it would add noise here.

jmcameron commented on docs/en/stock-management/accounting/index.md:
Per my comment above, you only have to access one account to get the stock (value) on hand with this system, right?

jniles replied:
Yes.  Added this text to the intro of this section for clarity.

jmcameron commented on docs/en/stock-management/accounting/index.md:
Does BHIMA support IFRS stock accounting?

jniles replied:
Yes, possibly better that OHADA stock accounting.

jmcameron reviewed (commented):
Given my limited understanding of OHADA, etc, you should take my comments with a grain of salt.

jniles committed:
docs(stock): flesh out COGS analysis

Incorporates feedback from @jmcameron to make the COGS analysis section
a bit richer.

jmcameron reviewed (approved):
I liked your updates!
